apps/nova/src/app/[locale]/(dashboard)/(admin)/(role-management)/users/page.tsx

```
@@ -0,0 +1,193 @@
+    if (role && role !== 'all') {
+      switch (role) {
+        case 'admin':
+          queryBuilder.eq('allow_role_management', true);
+          break;
+        case 'global_manager':
+          queryBuilder.eq('allow_manage_all_challenges', true);
+          break;
+        case 'challenge_manager':
+          queryBuilder.eq('allow_challenge_management', true);
+          break;
+        case 'member':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)